app/models/assignment.rb

```diff
@@ -365,6 +365,13 @@ def stage_deadline(topic_id = nil)
     due_date.nil? || due_date == 'Finished' ? due_date : due_date.due_at.to_s
   end
 
+  # E1834 Fall 18
+  def review_deadline(topic_id = nil, num_review_rounds)
```

Comment:
Not a good idea to put a method like this in the assignment class.  Is assignment going to have methods for all due-date types?  This seriously damages the cohesion of assignment.rb, which is a complicated class anyway.

Reply:
We have refactored the code and the method is removed from assignment class.